packages/headless-react/package.json

```diff
@@ -15,15 +15,20 @@
     "ssr"
   ],
   "license": "Apache-2.0",
-  "main": "dist/index.js",
+  "type": "module",
+  "exports": "./dist/index.js",
   "files": [
     "dist"
   ],
   "scripts": {
     "build": "nx build",
+    "clean": "rimraf dist",
```

Comment:
Thanks for rimraf 😄 we Windows users are often forgotten.
Please do include it in the dev dependencies of this pjson tho.